File: CodeWars/local_runner/plugins/model/Facility.java
package model;

public class Facility {
    private final long id;
    private final FacilityType type;
    private final long ownerPlayerId;
    private final double left;
    private final double top;
    private final double capturePoints;
    private final VehicleType vehicleType;
    private final int productionProgress;

    public Facility(
            long id, FacilityType type, long ownerPlayerId, double left, double top, double capturePoints,
            VehicleType vehicleType, int productionProgress) {
        this.id = id;
        this.type = type;
        this.ownerPlayerId = ownerPlayerId;
        this.left = left;
        this.top = top;
        this.capturePoints = capturePoints;
        this.vehicleType = vehicleType;
        this.productionProgress = productionProgress;
    }

    public long getId() {
        return id;
    }

    public FacilityType getType() {
        return type;
    }

    public long getOwnerPlayerId() {
        return ownerPlayerId;
    }

    public double getLeft() {
        return left;
    }

    public double getTop() {
        return top;
    }

    public double getCapturePoints() {
        return capturePoints;
    }

    public VehicleType getVehicleType() {
        return vehicleType;
    }

    public int getProductionProgress() {
        return productionProgress;
    }
}

File: CodeWars/Model/Facility.cs
using System;

namespace Com.CodeGame.CodeWars2017.DevKit.CSharpCgdk.Model {
    public class Facility {
        private readonly long id;
        private readonly FacilityType type;
        private readonly long ownerPlayerId;
        private readonly double left;
        private readonly double top;
        private readonly double capturePoints;
        private readonly VehicleType? vehicleType;
        private readonly int productionProgress;

        public Facility(long id, FacilityType type, long ownerPlayerId, double left, double top, double capturePoints,
                VehicleType? vehicleType, int productionProgress) {
            this.id = id;
            this.type = type;
            this.ownerPlayerId = ownerPlayerId;
            this.left = left;
            this.top = top;
            this.capturePoints = capturePoints;
            this.vehicleType = vehicleType;
            this.productionProgress = productionProgress;
        }

        public long Id => id;
        public FacilityType Type => type;
        public long OwnerPlayerId => ownerPlayerId;
        public double Left => left;
        public double Top => top;
        public double CapturePoints => capturePoints;
        public VehicleType? VehicleType => vehicleType;
        public int ProductionProgress => productionProgress;
    }
}